26-register_registration/2461-creator-adds-a-contributor.feature

```diff
@@ -0,0 +1,56 @@
+Feature: Creator adds a Contributor
+
+    Scenario Outline: Creator opens the Add Contributor Dialog
+        Given Creator navigates to Contributors tab
+        And the Registration is a "<RegistrationType>"
+        When they click "Add Contributor"
+        Then they see the "Add Contributor" Dialog
+        And they see a dropdown with Contributor Types "<ContributorTypes>"
+        And they see a "Close" Button
+        And they see a "Create new Contributor" Button
+        And they see a "Add me as Contributor" Button
+        And they see a disabled "Add" Button
+        Examples:
+            | RegistrationType          | ContributorTypes                                                                     |
+            | BookAnthology             | Contact person, Related person, Researcher, Rights holder, Supervisor, Other         |
```

Comment:
Slår meg at for testingen sin del så ville det vært greit om Resource Type var med i tillegg til subtype
`| RegistrationType | RegistrationSubtype | ContributorTypes |`
 | Book                    | BookAnthology        | Contact person, Related person, Researcher, Rights holder, Supervisor, Other |`